content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -0,0 +1,197 @@
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notRequestPrincipals: ["*"]
+EOF
+{{< /text >}}
+
+### Require different JWT issuer per host
+
+JWT validation is common on the ingress gateway and you may want to require different JWT issuers for different
+hosts, you can use the authorization policy for finer grained JWT validation in addition to the request authentication.
+
+Use the following policy if you want to allow requests only to the listed hosts and reject requests to any other hosts:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # the JWT token must have issuer with suffix "@example.com"
+        requestPrincipals: ["*@example.com"]
+    - source:
+        hosts: [".another.org", "*.another.org"]
+        # the JWT token must have issuer with suffix "@another.org"
+        requestPrincipals: ["*@another.org"]
+{{< /text >}}
+
+Alternatively, use the following policy if you want to require JWT validation only to the listed hosts but not for any
+other hosts:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # the JWT token must have issuer with suffix "@example.com"
+        notRequestPrincipals: ["*@example.com"]
+    - source:
+        hosts: [".another.org", "*.another.org"]
+        # the JWT token must have issuer with suffix "@another.org"
+        notRequestPrincipals: ["*@another.org"]
+{{< /text >}}
+
+### Namespace isolation
+
+You want to block all traffic from outside the namespace `foo`, in other words, isolate the namespace `foo` from other
+namespaces. This requires to enable mTLS first.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: foo-isolation
+  namespace: foo
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        namespaces: ["foo"]
+{{< /text >}}
+
+If you want to make sure the policy can not be bypassed by other `ALLOW` policies, you can use the `DENY` policy to
+achieve the result:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ns-isolation
+  namespace: foo
+spec:
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notNamespaces: ["foo"]
+{{< /text >}}
+
+### Namespace isolation with ingress exception
+
+You want to block all traffic from outside the namespace `foo` except the traffic from the ingress gateway.
```

Comment:
The allow policy is not block unless you have an existing "allow-nothing" policy.

Reply:
the deny-by-default behavior is enabled as long as there is any ALLOW policy, but not necessary a "allow-nothing" policy. So by applying the following ALLOW policy, anything not allowed will be denied by default.